File: chroma-league/src/main/java/com/bonepl/chromaleague/hud/animations/EnemyInfernalDragonKillAnimation.java
package com.bonepl.chromaleague.hud.animations;

import com.bonepl.chromaleague.rest.eventdata.model.DragonType;

import static com.bonepl.chromaleague.hud.PredefinedKeySets.BLACKWIDOW_FUNCTIONAL;

public class EnemyInfernalDragonKillAnimation extends StaticPartialBlinkingAnimation {
    public EnemyInfernalDragonKillAnimation() {
        super(BLACKWIDOW_FUNCTIONAL, 8, DragonType.INFERNAL.getColor());
    }
}

File: src/hud/animations/EnemyInfernalDragonKillAnimation.ts
import {DragonType} from '../../rest/eventdata/DragonType.js';
import {EnemyDragonKillBlinkingAnimation} from './EnemyDragonKillBlinkingAnimation.js';

export class EnemyInfernalDragonKillAnimation extends EnemyDragonKillBlinkingAnimation {
    constructor() {
        super(DragonType.INFERNAL);
    }
}
